wrangler-core/src/main/java/io/cdap/wrangler/utils/SqlTypeExpGenerator.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright © 2017-2019 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+package io.cdap.wrangler.utils;
+
+/**
+ * Utility class that returns a string of SQL expression for the given data type.
+ */
+public final class SqlTypeExpGenerator {
+
+    public static String getColumnTypeExp(String toType, String column, int scale) {
```

Comment:
Change this to have just one function and keep all the logic here. Mark the 3rd argument scale as @Nullable and you can pass in a null here. Then change the if-else logic in SetType where this method is called